src/documentation/Usage/UsageCss.stories.mdx

```diff
@@ -11,66 +11,95 @@ import { linkTo } from '@storybook/addon-links';
   }}
 />
 
-Each components requires css. You can import the css/scss from npm or use it from the CDN.
+> WFP front-end technology standards
+>
+> The WFP UI Kit uses [React](https://reactjs.org/) as the underlying JS framework, in line with the [WFP official guidelines for software development](https://newgo.wfp.org/documents/software-solutions-technology-stack). Therefore, **all WFP applications should use it**, together with the UI Kit. The only **exceptions** are off-the-shelf solutions (e.g. Saas) and/or non-apps (e.g. the external corporate website, and/or basic landing pages used for fundraising, etc).
 
-### React import
+> The following guidelines assume that you have already installed the Kit. If you haven't done it yet, please check our [step-by-step guide on how to install the Kit](https://uikit.wfp.org/docs/index.html?path=/docs/getting-started-installing-the-kit--page).
 
-You can also import the css directly from the npm package.
+There are **different ways you can use the Kit** in your applications.
 
-#### Import css via react
+## Usage with React
+**Default and recommended usage.** You can use a component of the Kit like any other React component: first, import it through the `import` function, use it with the `<Nameofcomponent></Nameofcomponent>` syntax.
```

Comment:
I would remove the last sentence and maybe write something like this:
All components are available as modules. For example `import { Button, Link } from '@wfp/ui'`

Reply:
Thanks @Utzel-Butzel, sounds good, will update it as suggested and use the conventional commit message.